### PR TITLE
added google assistant

### PR DESCRIPTION
**Pull request by Console32**
Closes https://github.com/ScoopInstaller/Extras/issues/8173

- [ x ] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
If you append a `#/dl.7z` at the end of URL, Scoop will extract it like a normal archive

**Comment by Console32** [comment]
There is no dl.7z available to download for this solution. Could not find any way to unzip the NSIS installer as well. /portable installation looks like the way to go from other manifests.

**Comment by rashil2000** [comment]
> If you append a `#/dl.7z` at the end of URL, Scoop will extract it like a normal archive

Just try it. You will understand what I mean. Remove the installer field

**Comment by Console32** [comment]
Just unziping the installer does not create the required artifacts, installer seems to be mandatory

**Comment by rashil2000** [comment]
Which artifacts are missing? You can create shortcuts using the `shortcuts` field

**Comment by Console32** [comment]
@rashil2000 the resulting folder does not contain the Google Assist executable, along with other missing parts. Could you elaborate what we are trying to do here ? Is it bad to use the installer?

**Comment by rashil2000** [comment]
See https://github.com/ScoopInstaller/Extras/blob/master/bucket/tusk.json

**Comment by Console32** [comment]
@rashil2000, thanks done

File: bucket/g-assist.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    ],
+    "checkver": "github",
+    "autoupdate": {
+                "architecture": {
+            "64bit": {
+                "url": "https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/releases/download/v$version/Google-Assistant-Setup-$version.exe#/dl.7z"
+            }
+        }
+    }
+}
```
Comment (rashil2000) [inline bucket/g-assist.json:24]:
```suggestion
{
    "version": "1.0.0",
    "description": "Unofficial desktop client for Google Assistant",
    "homepage": "https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client",
    "license": "Apache-2.0",
    "architecture": {
        "64bit": {
            "url": "https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/releases/download/v1.0.0/Google-Assistant-Setup-1.0.0.exe#/dl.7z",
            "hash": "c5acde8b3bd0db2215cb03ff817b95cd5d1de9888f31a8b02091fdf979a36101"
        }
    },
    "pre_install": "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\Uninst*\" -Force -Recurse",
    "shortcuts": [
        [
            "Google Assistant.exe",
            "Google Assistant"
        ]
    ],
    "checkver": "github",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/releases/download/v$version/Google-Assistant-Setup-$version.exe#/dl.7z"
            }
        }
    }
}
```

**Pull request merged**